chain/chunks/src/shards_manager_actor.rs

```diff
@@ -2091,9 +2094,10 @@ impl ShardsManagerActor {
             "type" = <&'static str>::from(&request)
         )
         .entered();
+        let me = self.validator_signer.get().map(|signer| signer.validator_id().clone());
```

Comment:
nit: Maybe add a comment? Not sure if it's better placed here or where the field is declared in the struct.  Something like:
```
Lock the value of mutable validator signer for the duration of the request to 
ensure consistency. Please note that the locked "me" should not be stored 
anywhere or passed through the thread boundary. 
```

Reply:
There already exist `me` in multiple place in the codebase, often taken as argument, so I updated fields where mutable `validator_signer` is declared: https://github.com/near/nearcore/pull/11400/commits/b2646ba17924d23c4f75fc345a52d3c33cce8379